docs/CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ The CHANGELOG for the current development version is available at
 [https://github.com/rasbt/biopandas/blob/main/docs/sources/CHANGELOG.md](https://github.com/rasbt/biopandas/blob/main/docs/sources/CHANGELOG.md).
 
 
+### 0.5.2 (27-03-2023)
```

Comment:
0.4.1 is still the latest release so these recent changes should actually go under 0.5.0dev

Reply:
Ok yeah mb I'll change that